pkg/services/ngalert/provisioning/contactpoints.go

```diff
@@ -23,21 +22,22 @@ import (
 )
 
 type ContactPointService struct {
-	amStore           AMConfigStore
+	config            *alertmanagerConfigStoreImpl
```

Comment:
Should these use the interface created in config.go (`alertmanagerConfigStore`) rather than the concrete impl directly?

Reply:
Currently, this is not trivial as many tests access the underlying store directly. However, in future PRs we'll gradually migrate to the fake of the upper store abstraction.